docs/view_components/create_scenario.rst

```diff
@@ -0,0 +1,59 @@
+..
+    one can then cross link to this view component by using
+    :ref:`create_scenario-label`
+
+Create Scenario
+---------------
+
+Create scenario view-component enables the user to create a new scenario object where the energy system can be implemented.
+
+Attributes
+^^^^^^^^^^
+
+**Combination of a text field and Create button**
+    The user can create a new scenario through this.
+
+    Requirements index: 1
+
+**Each scenario gets an unique ID**
+    Alphanumeric string, can be used to reference the scenario object in other locations.
+
+    Requirements index: 1
+
+Actions
+^^^^^^^
+
+1. Allows user to navigate through the file system either graphically or through text inputs
+2. Allows user to name the scenario object and write it to a chosen location in the computer's file system
+
+Requirements
+^^^^^^^^^^^^
+
+1. A string containing the correct file path that leads to the folder that holds the scenario file(s)
+2. After the user creates a scenario, they must be able to create another scenario following the same procedure, through this view-component
+3. When a new scenario is created, it should appear in the the list of available scenarios within the project in the scenarios view
```

Comment:
you can make references :ref:`project-label`, same for scenario view

Reply:
but where does `:ref:project-label `go ? in place of the word 'project' in line 34?

Reply:
where else ? :coffee: